lib/twingly/url.rb

```diff
@@ -193,6 +200,14 @@ def <=>(other)
       self.to_s <=> other.to_s
     end
 
+    def eql?(other)
+      self.hash == other.hash
```

Comment:
should we check that `other` is if same type too?

Reply:
Yeah, or maybe we can delegate it somehow, hmm.  But yeah that sounds reasonable.

Reply:
Implemented in https://github.com/twingly/twingly-url/pull/129/commits/d30bbd8324ad6bbd5622e88268da87f5714aeb55